src/actions/formatSelection.ts

```diff
@@ -47,24 +47,49 @@ export const formatSelectionActionCreator =
           if (elementColor && isDifferentColor) return true
           return false
         }
+
+        /** Function to collect tag names without significant attributes. */
+        const collectTagsWithoutAttributes = (text: string, pattern: RegExp): string[] =>
+          Array.from(text.matchAll(pattern))
+            // Filter out tags that lack meaningful attributes
+            .filter(([, , attributes]) => {
+              const meaningfulAttributes = ['style=', 'class=', 'color=', 'background-color=']
```

Comment:
We already have `ALLOWED_ATTR`, so let's use those. We want to avoid hardcoding values as if they are independent, when they clearly are tightly coupled to other parts of the app that handle parsing, stripping, and formatting.

https://github.com/cybersemics/em/blob/6e4fa5311c6b9e5ddb4baa50e352d33aa58b2f8b/src/constants.ts#L279